lib/closex/mock_client/fixtures/create_opportunity.json

```diff
@@ -0,0 +1,23 @@
+{
```

Comment:
Is it different from `opportunity.json`?

Reply:
Same as with `lead.json` and `create_lead.json`.